test/translates_test.rb

```diff
@@ -206,6 +206,14 @@ def test_with_translation_relation
     end
   end
 
+  def test_with_translation_when_ambiguous_column
+    p = Post.create!(:title_translations => { "en" => "Alice in Wonderland", "fr" => "Alice au pays des merveilles" })
+    Tag.create!(:title_translations => { "en" => "A Tag", "fr" => "Un tag" }, post: p)
+    I18n.with_locale(:en) do
+      assert_equal p.title_en, Post.tagged("A Tag").first.try(:title)
```

Comment:
```suggestion
    post = Post.create!(:title_translations => { "en" => "Alice in Wonderland", "fr" => "Alice au pays des merveilles" })
    Tag.create!(:title_translations => { "en" => "A Tag", "fr" => "Un tag" }, post: post)
    I18n.with_locale(:en) do
      assert_equal post.title_en, Post.tagged("A Tag").first.title
```

Reply:
I'd be happy to make that change, but I originally went with `p` because that's what was used in all the tests. I think a change like that should be a separate PR and fix update all existing tests in one pass. What do you think?